src/User32.Desktop/User32.cs

```diff
@@ -68,6 +69,16 @@ public static partial class User32
         [DllImport(nameof(User32), SetLastError = true)]
         public static extern int GetWindowLong(IntPtr hWnd, WindowLongIndexFlags nIndex);
 
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Auto)]
+        public static extern int GetClassName(IntPtr hWnd, StringBuilder lpClassName, int nMaxCount);
```

Comment:
Yes, that makes sense. I guess we shouldn't forbid use of `StringBuilder` in at least many cases. I've found in most of the PInvoke.* code in this project so far that staying closer to the native side tends to pay off in flexibility later though. I'm just not sure about the balance between `StringBuilder` and `char*` is in ever case yet. 
I trust you saw in the version I merged, I replaced this with `char*` and then added [a helper overload](https://github.com/AArnott/pinvoke/blob/master/src/User32.Desktop/User32.Helpers.cs#L38) that is super easy to use (easier than `StringBuilder`). I hope you find it satisfactory.